io.github.Figma_Linux.figma_linux.appdata.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally you should maintain it in upstream repo.

Reply:
Yes, sure